go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/PuerkitoBio/goquery v1.8.1
 	github.com/avast/retry-go v3.0.0+incompatible
 	github.com/aws/aws-sdk-go v1.44.195
-	github.com/aws/karpenter-core v0.27.3
+	github.com/aws/karpenter-core v0.27.4-0.20230516213910-fd3f7d15c696
```

Comment:
Should these pin the version `v0.27.4` directly?

Reply:
I wanted to run the e2e tests first... but seems snapshot doesn't work when not destined for main :(

Reply:
Wait, really? That's super weird :(